roles/edpm_swift/tasks/configure.yml

```diff
@@ -25,6 +25,15 @@
   tags:
     - edpm_users
 
+- name: Allow unprivileged rsync use port 873
+  ansible.posix.sysctl:
+    name: net.ipv4.ip_unprivileged_port_start
```

Comment:
Hmm.. AFAIR this not a security best practice as it will allow user processes to bind to any ports >=873. Can we use capabilities instead or run rsync with escalated previleges in archive mode(-a)?